scripts/pre_commit_linter.py

```diff
@@ -2173,6 +2183,12 @@ def _check_bad_patterns(self):
                         print ''
                         total_error_count += 1
 
+                for regexp in BAD_PATTERNS_REGEXP:
+                    if _check_bad_pattern_in_file(
+                            filepath, file_content, regexp):
+                        failed = True
+                        total_error_count += 1
```

Comment:
@apb7, `total_error_count` is the number of files with bad patterns? (that's also not true though) or it's the number of failed patterns?

(I mean in one file there can be multiple errors for a single bad_pattern regexp.)

Reply:
I think it's the total number of failed patterns (for reference, please see other regexp checks). Thanks!